concepts/05 Widgets/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,264 @@
+To submit the Form, wrap it in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. Then add the [button](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/ButtonItem/) item and set its **buttonOptions**.[useSubmitBehavior](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#useSubmitBehavior) property to **true**. You can submit the Form data to the server only when all validation checks pass successfully.  
+
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#formContainer").dxForm({
+            formData: {
+                name: "John Heart",
+                position: "CEO",
+                hireDate: new Date(2012, 4, 13),
+                officeNumber: 901,
+                phone: "+1(213) 555-9392",
+                skype: "jheart_DX_skype",
+                email: "jheart@dx-email.com",
+                notes: "John has been in the Audio/Video industry since 1990."
+            },
+            colCount: 2,
+            items: [{
+                dataField: "name",
+                isRequired: "true"
+            }, "position", "hireDate", {
+                dataField: "officeNumber",
+                validationRules: [{
+                    type: "numeric",
+                    message: "This field should contain a number"
+                }] 
+            }, "notes", "phone", "skype", {
+                dataField: "name",
+                validationRules: [{
+                    type: "email",
+                    message: "This is not a valid Email"
+                }] 
+            }, {
+                itemType: "button",
+                buttonOptions: {
+                    text: "Submit the Form",
+                    useSubmitBehavior: true
```

Comment:
Again so many fields/options to demonstrate useSubmitBehavior.
It's difficult to find it in this code.